### ISSUE TITLE
All class default icons/sprays should also count as unlocked

### ISSUE THREAD

**Issue by KaeLL** [Js41637/Overwatch-Item-Tracker#240]
Because that's pretty much the only way to verify you haven't screwed up after marking over 400 or 500 items. You count them in the game, so the tracker should factor them too.

**Comment by Js41637** [comment]
Not sure what you mean?

**Comment by KaeLL** [comment]
On the All Class section, default items (such as the 16 OWL teams' icons and OWL team logo sprays, and a few others) don't count towards the resulting number of marked icons/sprays on the tracker. For example, counting my icons in-game results in a difference of 16 more than what it's shown in the tracker. In-game, I have 360, but the tracker shows only 344, and the difference is in the 16 OWL teams' icons, tagged as Default on the tracker. The same goes for sprays also tagged as Default.
I count them to check if I correctly marked what I own on the tracker, which is why not counting the Default icons/sprays towards the end result is confusing and unnecessary.

**Comment by Js41637** [comment]
This is just how default items work in game, I can't start counting them on the all hero page but not count them on the hero pages as it will make the counts incorrect.

**Comment by KaeLL** [comment]
Wait, what? Icons already don't count on hero pages, and that's how it should be. What are you talking about?

**Comment by Js41637** [comment]
Yeah cause it's an option in settings and aren't shown on the hero pages in the hero gallery, i'm saying that's how how default items work in game everywhere else though, default items being excluded from total. The way I see it is if by default you're progress should be `0`. If we count items that we know everyone owns then you can never start from 0 which just seems weird to me?

**Comment by Gonzom** [comment]
Blizzard just messed up a simple feature. They should have counted all (non-promotion) items, default or not. Seeing 11 skins in a list but having seeing the number 10 is confusing at the least. But sadly you should follow that as then the confusing factor will just increase when using the site and then having two different totals.

**Comment by KaeLL** [comment]
I too think it's weird, but I also believe it's a good enough reason for the change. Perhaps make it optional? Just because the original model (the game) is wonky, doesn't mean the very tool made to simplify things (the tracker) has to be wonky too, or that it has to be the only way.
Either way, it's up to you. I guess I'm the first bringing this up, and it seems there isn't much need or support for this, so yeah..
